### DISCUSSION TITLE
After installed OKay: WSL 2 - Ubuntu 20.04.2 in Windows 10 Pro YESTERDAY, I get an error when starting any virtual machine in VMware Workstation Pro 15.x.

### DISCUSSION THREAD

**Discussion by lse123** [microsoft/WSL#6854] [Q&A]
### Windows Build Number

Microsoft Windows [Version 10.0.19042.928]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Distro Version

Ubuntu 20.04.2

### Repro Steps

https://answers.microsoft.com/en-us/windows/forum/apps_windows_10-winapps-appscat_tools/after-installed-wsl-2-get-an-error-when-starting/b54d92e8-b211-47d8-acb5-98753cd7fefd?tm=1619095675468

After installed OKay: WSL 2 - Ubuntu 20.04.2 in Windows 10 Pro YESTERDAY, I get an error when starting any virtual machine in VMware Workstation Pro 15.x...? well? I tried Vmware forums, but having bugs in their site and cannot log in to post...

Picture - screenshots here:   

https://yadi.sk/i/dXwoTn12TqwTJA   

https://yadi.sk/i/CGp3JhZGScXGsA

### Expected Behavior

Why Not open VMs in VMware Workstation Pro 15.x....?

### Actual Behavior

VMs in VMware Workstation Pro 15.x opened OK prior install WSL 2...Well?

**Comment by 0xbadfca11** [comment]
You must disable Nested Virtualization in VMware.
If you needs nested virtualization in VMware, you can't use WSL2 together.
Windows Hypervisor Platform is not support Nested Virtualization yet until 21H1. (Insider Preview's Windows Hypervisor Platform supports nested virtualization, but I'm not sure VMware supports it.) 
![image](https://user-images.githubusercontent.com/7390559/115876912-d79c1900-a481-11eb-958a-cb9c1e347d15.png)
It's actually very confusing, but since version 1809, the Windows Hypervisor Platform is automatically usable when Windows Hypervisor (includes Hyper-V, Virtualization-based Security, and WSL2) is enabled. There is no need to enable manually.
https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/android-emulator/hardware-acceleration#:~:text=On%20Windows%2010%20October%202018%20Update,automatically.

**Comment by lse123** [comment]
Yes, that worked
I disable all 3 textboxes of VT-x in the Vmware Workstation Pro 15.x and now VM boots OK under the Vmware Workstation Pro 15.x 
Thank you for your support!
